sdk/attestation/azure-security-attestation/inc/azure/attestation.hpp

```diff
@@ -2,7 +2,7 @@
 // Licensed under the MIT License.
 
 /**
- * @brief Includes all public headers from Azure Attestation SDK library.
+ * @brief Includes all public headers from Azure Attestation SDK library. THis
```

Comment:
@LarryOsterman, it looks like #5199 did not help with "`THis`".
What it does now, is it became case-sensitive for the additional dictionary of words that we specify, but not the common dictionary.

I suggest to revert #5199, because it rather added problems, with little benefit, in my opinion. The original issue is not solved :(